cmd/state-installer/cmd.go

```diff
@@ -249,6 +249,9 @@ func execute(out output.Outputer, cfg *config.Instance, an analytics.Dispatcher,
 	if err != nil {
 		return errs.Wrap(err, "Could not detect if State Tool is already installed.")
 	}
+	if installPath != params.path {
+		params.path = installPath
+	}
```

Comment:
Please add the logic for adding the branch earlier on, and modify `installPath`. By modifying `params.path` you're creating the potential for new logic paths that we aren't interested in for this fix.